one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in feature1 branch.
+
```

Comment:
Hey, it's really fantastic that you have used this line

Reply:
Thank you Nasir